Apromore-Core-Components/Apromore-Portal/src/main/resources/default-menus.json

```diff
@@ -76,6 +76,9 @@
             {
               "id": "org.apromore.plugin.portal.loganimation2.LogAnimationPlugin"
             },
+            {
+              "id": "org.apromore.plugin.portal.loganimation_ee.LogAnimationPlugin"
```

Comment:
Same as the above comment. Would be nice if the package structure could be something like `org.apromore.ee.plugin.portal.loganimation.LogAnimationPlugin`

Reply:
Done